lib/base.php

```diff
@@ -51,11 +51,14 @@ public static function init() {
 
         // register the log file
         $logLevel = self::$config->getValue('info', 'log_level', Logger::WARNING);
-        self::$logger->popHandler();
-        self::$logger->pushHandler(new StreamHandler(
-            self::path(self::$config->getValue('info', 'private_dir').'/mesamatrix.log'),
-            $logLevel
-        ));
+        $logPath = self::path(self::$config->getValue('info', 'private_dir').'/mesamatrix.log');
+        if (is_writable($logPath)) {
+            self::$logger->popHandler();
+            self::$logger->pushHandler(new StreamHandler($logPath, $logLevel));
+        } else {
```

Comment:
Ah sorry, it's just a habit. There are so many variations of code formatting that I have to deal with on a daily basis... :laughing: